static/app/components/replays/player/fastForwardBadge.tsx

```diff
@@ -13,9 +13,9 @@ type Props = {
 function FastForwardBadge({speed, className}: Props) {
   return (
     <Badge className={className}>
-      <FastForwardTooltip title={t('Fast forwarding')}>
+      <FastForwardTooltip title={t(`Fast forwarding at ${speed}x`)}>
+        {t('Fast forwarding through inactive')}
```

Comment:
```suggestion
        {t('Fast forwarding through inactivity')}
```

Reply:
Slight edit that makes more sense - I realize the GH issue I created missed this. 😄